src/Components/Dashboard/Dashboard.js

```diff
@@ -1,86 +1,99 @@
-import rainNightIcon from '../../assets/rain-night.png';
-import {convertToFahrenheit, convertWindToCardnialDirection, convertMsToMph, displayCorrectWeatherIcon} from '../../utilities.js';
+import filledFavIcon from '../../assets/Five_Pointed_Star_Solid.svg'
+import {convertToFahrenheit, convertWindToCardinalDirection, convertMsToMph, displayCorrectWeatherIcon, airQualityMessages} from '../../utilities.js';
 
 function Dashboard({location, backToLandingPage}) {
+  const [isStarred, setIsStarred] = useState(false)
+
   const locationAqi = location.current.pollution.aqius;
   const tempInFahrenheit = convertToFahrenheit(location.current.weather.tp);
-  const windDirection = convertWindToCardnialDirection(location.current.weather.wd);
+  const windDirection = convertWindToCardinalDirection(location.current.weather.wd);
   const windMph = convertMsToMph(location.current.weather.ws);
   const weatherIconWithAltText = displayCorrectWeatherIcon(location.current.weather.ic);
 
-//CREATE METHOD THAT HANDLES FAV ICON CLICK (SAVE TO FAV LOCATIONS/LOCAL STORAGE)
-//CREATE METHOD THAT HANDLES AQI NUMBER CLICK TO BRING YOU TO INFORMATION PAGE
-
-  const airQualityMessages = (aqi) => {
-    if (aqi <= 50) {
-      return [
-        'GOOD',
-        'Air quality is satisfactory, and air pollution poses little or no risk.'
-      ];
-    }
-    else if (aqi > 50 && aqi <= 100) {
-      return [
-        'MODERATE',
-        'Air quality is acceptable. However, there may be a risk for some people, particularly those who are unusually sensitive to air pollution.'
-      ];
-    }
-    else if (aqi > 100 && aqi <= 150) {
-      return [
-        'UNHEALTHY FOR SENSITIVE GROUPS',
-        'Members of sensitive groups may experience health effects. The general public is less likely to be affected.'
-      ];
-    }
-    else if (aqi > 150 && aqi <= 200) {
-      return [
-        'UNHEALTHY',
-        'Some members of the general public may experience health effects; members of sensitive groups may experience more serious health effects.'
-      ];
+const toggleStar = () => {
+  setIsStarred(isStarred => !isStarred)
+  const jsonLocation = JSON.stringify(location)
+    if(localStorage.getItem(`${location.city}`) === null) {
+      localStorage.setItem(`${location.city}`, jsonLocation)
     }
-    else if (aqi > 200 && aqi <= 300) {
-      return [
-        'VERY UNHEALTHY',
-        'Health alert: The risk of health effects is increased for everyone.'
-      ];
-    } else {
-      return [
-        'HAZARDOUS',
-        'Health warning of emergency conditions: everyone is more likely to be affected.'
-      ]
+    if(isStarred) {
+      localStorage.removeItem(`${location.city}`)
     }
-  }
+}
+
+useEffect(() => {
+  setIsStarred(false)
+  const localStorageKeys = Object.keys(localStorage)
+    if (localStorageKeys.includes(location.city)) {
+      setIsStarred(true)
+    } 
+}, [location])
+
+  // const airQualityMessages = (aqi) => {
```

Comment:
I believe we can get rid of this commented out function now that it lives in the utilities file?